package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "asyncios",
+  "version": "0.0.0",
+  "description": "A wrapper for promise-based library Axios into a crocks Async",
+  "main": "index.js",
+  "repository": "git@github.com:articulate/asyncios.git",
+  "author": "engineering@articulate.com",
+  "license": "MIT",
+  "private": false,
+  "scripts": {
+    "lint": "eslint .",
+    "test": "karma start --single-run",
+    "test:watch": "karma start"
+  },
+  "dependencies": {
+    "axios": "^0.19.0",
+    "crocks": "^0.9.4"
```

Comment:
Any reason to not upgrade this while we are in there.
I am guessing this was to keep bundle sizes down on project that use this dependency.

Reply:
Correct, which is quickly becoming a moot point. We may want to bang our heads on this tomorrow.

Reply:
Actually.. i'm going to bump this... Stand by...